eiffel-vocabulary/EiffelTestCaseStartedEvent.md

```diff
@@ -17,10 +17,15 @@ __Type:__ String
 __Required:__ Yes  
 __Description:__ The unique identity of the executed test case.
 
+#### data.testCase.version
+__Type:__ String  
+__Required:__ No  
+__Description:__ The unique version of the executed test case identity.
```

Comment:
Perhaps append that "Where this property is not used it is assumed that test cases are not version controlled."